doc/source/whatsnew/v0.21.0.txt

```diff
@@ -160,6 +160,7 @@ I/O
 ^^^
 
 - Bug in :func:`read_csv` in which non integer values for the header argument generated an unhelpful / unrelated error message (:issue:`16338`)
+- Bug in ``DataFrame.to_html()`` with ``notebook=True`` where DataFrames with named indexes or non-MultiIndex indexes had undesired horizontal or vertical alignment for column or row labels, respectively (:issue:`16792`) 
```

Comment:
indexes --> indices

", respectively" --> " respectively"

Reply:
indices/indexes is actually both correct :-) 
http://grammarist.com/usage/indexes-indices/